[Bug]: AssertionError when using a list of local files as input_dir

### What happened?

Using a list of local files in  fastdup.create( input_dir=input_files, ... ) causes an Assertion Error:
`AssertionError: Failed to extract any files from list`

**I will provide a fix in a Fork and a Pull Request.**

### What did you expect to see?

 ########################################################################################
Dataset Analysis Summary: 

    Dataset contains 100 images
.....

### What version of fastdup were you runnning on?

0.927

### What version of Python were you running on?

Python 3.9

### Operating System

Ubuntu

### Reproduction steps


Install HuggingFace library of datasets
`pip install datasets`

```
import datasets
import fastdup
from pathlib import Path

# load food101 dataset with images as local files paths
dataset = datasets.load_dataset("renumics/food101-enriched", split="all")
df = dataset.to_pandas()

# get the first 100 image local paths
input_files = [str(x) for x in df['image'].iloc[:100]]
print(input_files[:5])
assert all([Path(x).exists() for x in input_files])

# create a fastdup with the input files and run it
fd = fastdup.create(work_dir="/tmp/fastdub_workdir", input_dir=input_files)
fd.run(ccthreshold=0.9) 
```

### Relevant log output

```shell
AssertionError                            Traceback (most recent call last)
Cell In[8], line 16
     14 # create a fastdup with the input files and run it
     15 fd = fastdup.create(work_dir="/tmp/fastdub_workdir", input_dir=input_files)
---> 16 fd.run(ccthreshold=0.9) 

File ~/playbook/.venv/lib/python3.9/site-packages/fastdup/engine.py:157, in Fastdup.run(self, input_dir, annotations, embeddings, subset, data_type, overwrite, model_path, distance, nearest_neighbors_k, threshold, outlier_percentile, num_threads, num_images, verbose, license, high_accuracy, cc_threshold, **kwargs)
    154     fastdup_func_params['model_path'] = model_path
    155 fastdup_func_params.update(kwargs)
--> 157 super().run(annotations=annotations, input_dir=input_dir, subset=subset, data_type=data_type,
    158             overwrite=overwrite, embeddings=embeddings, **fastdup_func_params)

File ~/playbook/.venv/lib/python3.9/site-packages/fastdup/sentry.py:134, in v1_sentry_handler..inner_function(*args, **kwargs)
    132 except Exception as ex:
    133     fastdup_capture_exception(f"V1:{func.__name__}", ex)
--> 134     raise ex

File ~/playbook/.venv/lib/python3.9/site-packages/fastdup/sentry.py:128, in v1_sentry_handler..inner_function(*args, **kwargs)
    126 try:
    127     start_time = time.time()
--> 128     ret = func(*args, **kwargs)
    129     fastdup_performance_capture(f"V1:{func.__name__}", start_time)
    130     return ret
...
    211             assert False, f"Unknown file type encountered in list: {f}"
--> 213 assert len(files), "Failed to extract any files from list"
    214 return files

AssertionError: Failed to extract any files from list
```


### Attach a screenshot [Optional]

_No response_

### Contact Details [Optional]

--- a/fastdup/utils.py
+++ b/fastdup/utils.py
@@ -2,6 +2,7 @@ import os
 import glob
 import random
 import platform
+from pathlib import Path
 from fastdup.definitions import *
 from datetime import datetime
 from fastdup.sentry import fastdup_capture_exception
@@ -192,8 +193,8 @@ def expand_list_to_files(the_list):
     files = []
     found = False
     for f in the_list:
-        if isinstance(f, str):
-            if f.startswith("s3://") or f.startswith("minio://"):
+        if isinstance(f, str) or isinstance(f, Path):
+            if isinstance(f, str) and (f.startswith("s3://") or f.startswith("minio://")):
                 for suffix in IMAGE_SUFFIXES:
                     if f.lower().endswith(suffix):
                         found = True

